index.html

```diff
@@ -3,9 +3,47 @@
   <head>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300italic,300,600,800' rel='stylesheet' type='text/css'>
   </head>
   <body>
     <header>
-    </header>
+     <h1>Relaxr</h1>
+     <h2>Get piece of mind with a single tap</h2>
+     <h3 class="btn">Get it Now</h3>
+    </header> 
+   <section>
+   <h4>Benefits</h4>
+   <p>The perfect personal assistant. Relaxr does your</p>
+   <p>job for you so you can enjoy life as it is meant to</p>
+   <p>be.</p>
+   </section>
+<section id="list">
+<ul>
+	<li>Schedule meetings for you</li>
+	<li>Excel automation to complete your work for you</li>
+	<li>Responds to emails on your behalf</li>	
+	<li>Does all your work for you with our revolutionary AI technology</li>
+</ul>
+</section> 
+<section class="quote">
+<blockquote class="quote"><p class="quote">&#34Relaxr changed my life. I&#39ve been able </p>
+<p class="quote">to travel the world, spend limited time</p>
```

Comment:
You don't need to add multiple `<p>` tags here. You can simple have the text insite the blockquote and let it wrap naturally.